src/lib/strapi.ts

```diff
@@ -90,7 +92,7 @@ export class Strapi {
     try {
       const response: AxiosResponse<T> = await this.axios.request<T>({
         method,
-        url,
+        url: join("/api", url),
```

Comment:
You may want to make this configurable, yes this will be the default but users could change it.

Reply:
Done